azkaban-common/src/test/java/azkaban/utils/EmailerTest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package azkaban.utils;
+
+import azkaban.executor.ExecutableFlow;
+import azkaban.flow.Flow;
+import azkaban.project.DirectoryFlowLoader;
+import azkaban.project.Project;
+import azkaban.test.executions.TestExecutions;
+import com.google.common.io.Resources;
+import org.apache.log4j.Logger;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+
+public class EmailerTest {
+
+    String host = "smtp.domain.com";
+    int mailPort = 25;
+    String sender = "somebody@domain.com";
+    String user = "somebody@domain.com";
+    String password = "pwd";
+
+    String failAddr = "receive@domain.com";
+    List<String> failAddrList = new ArrayList<String>();
+
+    private Project project;
+    private Props props;
+
+
+
+
+    @Before
+    public void setUp() throws Exception {
+        failAddrList.add(failAddr);
+        project = new Project(11, "myTestProject");
+        Logger logger = Logger.getLogger(this.getClass());
+
+        props =  createMailProperties();
+        DirectoryFlowLoader loader = new DirectoryFlowLoader(props, logger);
+        loader.loadProjectFlow(project, TestExecutions.getFlowDir("embedded"));
+        Assert.assertEquals(0, loader.getErrors().size());
+        project.setFlows(loader.getFlowMap());
+        project.setVersion(123);
+    }
+
+
+
+
+    /**
+     * test send email
```

Comment:
I saw that this test is no longer ignored. How does it work? Where is the email actually delivered?

I think it is a fine idea to create a test to make end to end testing email sending feature easier. In that case it should be clearly documented how that test should be customized to fit the test environment and run.